testnets/uniontestnet/chain.json

```diff
@@ -33,19 +33,18 @@
   "peers": {
     "seeds": [
       {
-        "id": "b37de4c50e26f7cde4c7b6ce06046a6693ffef2c",
-        "address": "union.testnet.6.seed.poisonphang.com:26656",
-        "provider": ""
-      },
-      {
-        "id": "2812a4ae3ebfba02973535d05d2bbcc80b7d215f",
-        "address": "union-testnet-seed.itrocket.net:23656",
+        "id": "c2bf0d5b2ad3a1df0f4e9cc32debffa239c0af90",
+        "address": "testnet.seed.poisonphang.com:26656",
         "provider": ""
```

Comment:
Can we add a provider name here? Is it "Poison Phang"?

Reply:
"poisonphang" is fine